app/src/main/java/io/appium/uiautomator2/core/AccessibilityNodeInfoHelpers.java

```diff
@@ -36,6 +36,8 @@
  * This class contains static helper methods to work with {@link AccessibilityNodeInfo}
  */
 public class AccessibilityNodeInfoHelpers {
+	
```

Comment:
please add a comment here with a link to the original UIA2 bug, which enforces us to have this workaround